docs/conf/portland/2025/news/welcome.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: June 14, 2024
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+We're excited to announce the dates for our Portland 2025 conference!
+
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We'll be back at Revolution Hall for another year,
+full of great talks, networking, and community building.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+**We plan to launch our CFP in November**,
```

Comment:
Pending CFP edits

Reply:
I updated this to not mention a specific ate.